src/main/java/com/baran/charactersheetcreator/configuration/UserDetails.java

```diff
@@ -0,0 +1,29 @@
+package com.baran.charactersheetcreator.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.crypto.factory.PasswordEncoderFactories;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.provisioning.InMemoryUserDetailsManager;
+
+@Configuration
+public class UserDetails {
+    @Bean
+    public InMemoryUserDetailsManager userDetailsService() {
+        PasswordEncoder encoder = PasswordEncoderFactories.createDelegatingPasswordEncoder(); // password encoder is necessary since spring security 5: https://www.baeldung.com/spring-security-5-default-password-encoder
+        org.springframework.security.core.userdetails.UserDetails admin = User.withUsername("admin")
+                .password(encoder.encode("1234"))
```

Comment:
Passwords shouldn't be hardcoded in the code itself.  

Even placeholder passwords should in your application properties and read from there:
https://github.com/BaranselKarakas/character-sheet-creator/blob/main/src/main/resources/application.properties

Reply:
like this?
3acda9eb35f4c3e726ba1f87e29eb5ad64f346e1